proto-public/pbacl/acl.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
+// 	protoc-gen-go v1.31.0
```

Comment:
Here and below: all comment changes generated by `make proto` that came with upgrade, necessary to satisfy linter (and needed bc we changed the `proto-gen-go` version)